msal/src/Microsoft.Identity.Client/ClientApplicationBase.cs

```diff
@@ -187,6 +187,11 @@ internal TokenCache UserTokenCache
         /// </summary>
         public bool ValidateAuthority { get; set; }
 
+        /// <summary>
+        /// 
+        /// </summary>
```

Comment:
" Used to set the flag for AAD extended lifetime" ?

Reply:
@jmprieur to review public comments

Reply:
What about:
ExtendedLifeTimeEnabled is a Boolean that first party applications (read Office) can set to true in case when the STS has an outage, to be more resilient. See also <see cref="P:AuthenticationResult.IsExtendedLifetimeToken"/>